ci-operator/step-registry/rosa-qe/sts/cluster/nightly/rosa-qe-sts-cluster-nightly-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: rosa-qe-sts-cluster-nightly
+  steps:
+    env:
+      ACCOUNT_ROLES_PREFIX: "rosa-sts-nightly"
+      CLUSTER_NAME: "sdq-nrosa-sts"
```

Comment:
all cluster names are project's name, e.g. `ci-op-lh1tnz6x`, so I think we do not need to set CLUSTER_NAME as a parameter

Reply:
yes, but the reason for setting the cluster name is that there are some persons from ocp qes who want to have a way to distinguish the rosa cluster from the other cluster, defining a cluster name is the easiest way to achieve this requirement.

Reply:
Think again. Remove the cluster name defined in the workflow, if we want a named cluster, we should define it in the job config but not in the workflow.